src/components/EventCard/EventCard.tsx

```diff
@@ -11,39 +11,31 @@ const EventCard = (props: any) => {
   }
 
   return (
-    <Card className={classes.cardContainer}>
-      <CardActionArea className={classes.cardClick} onClick={cardClick}>
-        <CardMedia
-          className={classes.cardMedia}
-          component="img"
-          alt={props.event.name}
-          image={props.event.photoURL}
-          title={props.event.name}
-        />
-        <CardContent className={classes.cardContent}>
-          <Typography className={classes.subtitle} color="textSecondary" gutterBottom={true}>
-            {moment(props.event.start).format("ddd h:mma, MMM DD 'YY")}
-          </Typography>
-          <Typography
-            className={classes.title}
-            color="textPrimary"
-            variant="h5"
-            component="h2"
-            gutterBottom={true}
-          >
-            {props.event.name}
-          </Typography>
-          <Typography
-            className={classes.desc}
-            color="textSecondary"
-            variant="body2"
-            component="p"
-            noWrap={false}
-          >
-            {props.event.desc}
-          </Typography>
-        </CardContent>
-      </CardActionArea>
+    <Card className={classes.cardContainer} onClick={cardClick}>
```

Comment:
Fixed the ellipsis and sizing issue. Got  of CardActionArea

Reply:
I didn't know Card as an onClick attribute, nice.